src/main/java/com/ibm/crypto/plus/provider/ock/Digest.java

```diff
@@ -48,10 +50,6 @@ public final class Digest {
 
     static private int runtimeContextNum[];
 
-    class NativeResources {
-        private long digestId = 0;
-    }
-
     class ConcurrentLinkedQueueLong extends ConcurrentLinkedQueue<Long> {
```

Comment:
ConcurrentLinkedQueueLong contexts[] are also a partial Digest, though we should implements cloneable for this inner class as well?

Reply:
We don't need to clone that, because it is static and therefore not instance specific.

Reply:
right, miss the static